echo-rest/src/main/groovy/com/netflix/spinnaker/echo/config/RestProperties.groovy

```diff
@@ -41,6 +41,8 @@ class RestProperties {
     String url
     String username
     String password
+    Map<String, String> headers
+    String headersFile
```

Comment:
I assume the header content is in a file instead of directly in the config cause it's frequently going to have secrets? That's the only thing that made me think for a moment, the rest looks great.